modules/ROOT/pages/6.0-release-notes-commands-and-queries.adoc

```diff
@@ -0,0 +1,86 @@
+= TinyMCE 6.0 Commands and queries changes
+:navtitle: TinyMCE 6.0 Commands and queries changes
+:description: TinyMCE 6.0 Commands and queries changes
+:keywords: releasenotes, commands, queries
+
+This section documents the improvements and changes made to {productname} 6.0 Commands and queries.
+
+It also documents the {productname} 6.0 Commands that have been removed.
+
+
+* xref:commands-and-queries[Commands and queries]
+* xref:new-and-improved-commands-and-queries[New and improved commands and queries]
+* xref:changed-commands-and-queries[Changed commands and queries]
+* xref:removed-or-deprecated-commands-and-queries[Removed or deprecated commands and queries]
+
+
+// tag::commands-and-queries[]
+[[commands-and-queries]]
+== Commands and queries
+
+
+[[new-and-improved-commands-and-queries]]
+=== New and improved commands and queries
+
+
+[[mce-autocompleter-close-and-mce-autocompleter-reload]]
+==== `mceAutocompleterClose` and `mceAutocompleterReload`
+
+As part of moving `autocompleter` from the default `silver` theme to the {productname} core, two new commands are now available: `mceAutocompleterClose` and `mceAutocompleterReload`.
+
+[[mce-insert-table-dialog]]
+==== mceInsertTableDialog
+
+`mceInsertTableDialog` is the new command for invoking an _Insert Table_ dialog box.
+
+The previous command, `mceInsertTable`, no longer works to invoke an _Insert Table_ dialog box: use the `mceInsertTableDialog` command instead.
+
+The `mceInsertTable` command is still the command for inserting a table with specified properties directly and without asking for user input.
+
+
+[[changed-commands-and-queries]]
+=== Changed commands and queries
+
+
+[[editor]]
+==== Editor commands
+
+The Editor commands APIs no longer fallback to executing the browsers native command functionality.
+
+As well, the Editor query command APIs now return `false` or an empty string on removed {productname} editors.
+
+
+[[element-format]]
+==== `element_format`
```

Comment:
I noted this on another page too pretty sure, but this isn't a command I believe. It should be an option, just based on the name.

Reply:
Section moved from `6.0-release-notes-commands-and-queries.adoc` to `6.0-release-notes-options.adoc`.